views/segments.html

```diff
@@ -99,7 +99,7 @@
     'numberOfItemsSingular': __('1 item'),
     'numberOfItemsMultiple': __('%$1d items'),
     'allSendingPausedHeader': __('All sending is currently paused!'),
-    'allSendingPausedBody': __('Your key to send with MailPoet is invalid.'),
+    'allSendingPausedBody': __('Your [link]API key[/link] to send with MailPoet is invalid.'),
```

Comment:
@pavel-mailpoet What about moving  'allSendingPausedHeader', 'allSendingPausedBody' and 'allSendingPausedLink' from newsletters.html, subscribers.html and segments.html to layout.html? Or do they need to be on each file?

Reply:
It is fine to move them to `layout.html`. Good idea.